test/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -358,7 +359,9 @@ class CheckYourAnswersControllerSpec extends SpecBase with MockitoSugar with Bef
               .set(DateOfBirthPage, aDateOfBirth).success.value
               .set(AddressInUkPage, true).success.value
               .set(UkAddressPage, aUkAddress).success.value
-            val expectedRegistrationRequest = IndividualWithoutId("first", "last", aDateOfBirth, Address.fromUkAddress(aUkAddress))
+              .set(IndividualEmailAddressPage, aContact.emailAddress).success.value
+
+            val expectedRegistrationRequest = IndividualWithoutId("first", "last", aDateOfBirth, Address.fromUkAddress(aUkAddress), aContact)
```

Comment:
Same as above and please do the same for all the tests.

Reply:
Done.